test/new-e2e/pkg/e2e/suite.go

```diff
@@ -222,12 +222,19 @@ func (bs *BaseSuite[Env]) IsDevMode() bool {
 	return bs.params.devMode
 }
 
-func (bs *BaseSuite[Env]) init(options []SuiteOption, self Suite[Env]) {
+func (bs *BaseSuite[Env]) init(t *testing.T, options []SuiteOption, self Suite[Env]) {
+	devMode, err := runner.GetProfile().ParamStore().GetBoolWithDefault(parameters.DevMode, false)
+	if err != nil {
+		t.Logf("dev_mode parameter error : %s", err)
```

Comment:
```suggestion
		t.Errorf("Unable to get DevMode value, DevMode will be disabled, err: %v", err)
```

Reply:
I don't think we should have the test failed if the parameters parser have an issue
like E2E_DEV_MODE=working should failed as it's not a boolean

Reply:
The `GetXX` does not fail if parameter is not present, otherwise the default value would be useless.
But yes I agree you should not fail the test. I always forget `t.Error` fails the test.

However changing the message is, I think, an improvement.

Reply:
Update the suggestion !